### PR TITLE
Change Rosetta reconciliation to use hardcoded balance offset

### LINKED ISSUE #4482: Change Rosetta reconciliation to use hardcoded balance offset
### Description

The Rosetta reconciliation fails because consensus nodes don't currently respect the invariant that a balance file with timestamp `T` includes only transactions whose consensus time is `<= T`.

### Steps to reproduce

Run rosetta reconciliation against mainnet

### Additional context

_No response_

### Hedera network

mainnet

### Version

v0.64.1

### Operating system

_No response_

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

- Add fixed account balance file offset for mainnet and testnet
- Switch to flyway multi-arch docker image 
- Report 0 balance for blocks in which an alias account doesn't exist

**Related issue(s)**:

Fixes #4482 

**Notes for reviewer**:

Passed rosetta-cli check:data with one-month worth mainnet data from 07/21/22

The 0-balance for alias account not exist in block is to make rosetta-cli check:data happy... 

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by edwin-greene — APPROVED** [review]
Looks good to me.

File: hedera-mirror-rosetta/test/db/db.go [hunk 1]
```diff
@@ -234,7 +234,7 @@ func runFlywayMigration(pool *dockertest.Pool, network *dockertest.Network, para
 	// run the container with tty and entrypoint "bin/sh" so it will stay alive in background
 	options := &dockertest.RunOptions{
 		Repository: "flyway/flyway",
-		Tag:        "8.5-alpine",
+		Tag:        "9.3.1",
```
Comment (steven-sheehy) [inline hedera-mirror-rosetta/test/db/db.go:237]: Maybe just use `9` so we don't have to update it as often.

File: hedera-mirror-rosetta/app/persistence/transaction.go [hunk 2]
```diff
@@ -287,14 +293,18 @@ func (t tokenTransfer) getAmount() types.Amount {
...
+	types                           map[int]string
 }
 
 // NewTransactionRepository creates an instance of a TransactionRepository struct
-func NewTransactionRepository(dbClient interfaces.DbClient) interfaces.TransactionRepository {
-	return &transactionRepository{dbClient: dbClient}
+func NewTransactionRepository(dbClient interfaces.DbClient, network string) interfaces.TransactionRepository {
+	return &transactionRepository{
+		dbClient:                        dbClient,
+		firstFixedOffsetTimestampSqlArg: getFirstAccountBalanceFileFixedOffsetTimestampSqlNamedArg(network),
```
Comment (steven-sheehy) [inline hedera-mirror-rosetta/app/persistence/transaction.go:306]: Are you sure it's okay to do a query in the constructor? Won't that cause failure if db is down/starting? Probably better to do lazily in a method call.

File: hedera-mirror-rosetta/app/persistence/common.go [hunk 3]
```diff
@@ -20,10 +20,36 @@
...
+                               lateral (
+                                 select
+                                   case when consensus_timestamp >= @first_fixed_offset_timestamp then 53
+                                        else 0
+                                   end value
+                               ) fixed_offset
                              order by consensus_timestamp
                              limit 1`
 	genesisTimestampCte = " genesis as (" + genesisTimestampQuery + ") "
+	mainnet             = "mainnet"
```
Comment (steven-sheehy) [inline hedera-mirror-rosetta/app/persistence/common.go:38]: There's not a network enum somewhere we can reuse either in our code or in SDK?

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: hedera-mirror-rosetta/app/persistence/common.go (hunk 3 above)
Comment (xin-hedera) [reply hedera-mirror-rosetta/app/persistence/common.go:38]: go doesn't have enum...

File: hedera-mirror-rosetta/app/persistence/transaction.go (hunk 2 above)
Comment (xin-hedera) [reply hedera-mirror-rosetta/app/persistence/transaction.go:306]: the function just creates a sql.NamedArg struct for the network the rosetta server is configured with, and it's reused in relevant queries later on. It doesn't run any sql query itself.

File: hedera-mirror-rosetta/test/db/db.go (hunk 1 above)
Comment (xin-hedera) [reply hedera-mirror-rosetta/test/db/db.go]: changed to just `9`

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Pull request merged**